ADD: diff:ignoreChildren and diff:ignoreAttributes description to docs (#572)
bUnit-dev/bUnit: docs/site/docs/verification/semantic-html-comparison.md

  </header>
  ```

> - **Ignore children:** Use the `diff:ignoreChildren` attribute (no value is needed) to ignore all child nodes of an element and **not** its attributes. For example, to ignore all child nodes of the `h1` element, do the following:

Reviewer: ```suggestion
- **Ignore children:** Use the `diff:ignoreChildren` attribute (no value is needed) to ignore all child nodes/elements of an element. This does not include attributes. For example, to ignore all child nodes of the `h1` element, do the following:
```

  ```html
  <header>

>     <h1 diff:ignoreChildren>Hello world</h1>

Reviewer: ```suggestion
    <h1 diff:ignoreChildren>Hello <span>world</span></h1>
```
Lets add a child element as well to drive home the point.

  </header>
  ```

> - **Ignore all attributes:** Use the `diff:ignoreAttributes` attribute (no value is needed) to ignore all attributes of an element and **not** its child nodes. For example:

Reviewer: ```suggestion
- **Ignore all attributes:** Use the `diff:ignoreAttributes` attribute (no value is needed) to ignore all attributes of an element. For example:
```

No need to mention elements here. Attributes clearly doesnt include child elements and nodes.

  ```html
  <header>
    <h1 diff:ignoreAttributes>Hello world</h1>
  </header>
  ```

>   > [!NOTE]
>   > The `diff:ignoreChildren` and `diff:ignoreAttributes` attributes can be combined to **not** ignore an element, but to ignore all its attributes and its child nodes. For example:
> 
>   ```html
>   <header>
>     <h1 diff:ignoreChildren diff:ignoreAttributes>Hello world</h1>
>   </header>
>   ```

Reviewer: ```suggestion
  > [!NOTE]
  > The `diff:ignoreChildren` and `diff:ignoreAttributes` attributes can be combined to ignore all child nodes/element *and* attributes of an element, but still verify that the element itself exists. For example:

  ```html
  <header>
    <h1 diff:ignoreChildren diff:ignoreAttributes>Hello world</h1>
  </header>
  ```
```

Excellent point. I had not thought of this combination use case actually!
